CHANGELOG.md

```diff
@@ -1,6 +1,70 @@
-### citus v7.0.0 (unreleased) ###
+### citus v7.0.0 (August 28, 2017) ###
 
-* Replaces pg_dist_shard_placement metadata table with pg_dist_placement
+* Adds support for PostgreSQL 10
+
+* Drops support for PostgreSQL 9.5
+
+* Adds support for multi-row `INSERT`
+
+* Adds support for router `UPDATE` and `DELETE` queries with subqueries
+
+* Adds infrastructure to distributed deadlock detection
+
+* Adds maintenance daemon for distributed deadlock detection and 2PC recovery
+
+* Deprecates `enable_deadlock_prevention` flag
+
+* Adds support for partitioned tables
+
+* Adds support for creating `UNLOGGED` tables
+
+* Adds support for `SAVEPOINT`
+
+* Adds UDF `citus_create_restore_point` for taking distributed snapshots
+
+* Adds support for evaluating non-pushable `INSERT ... SELECT` queries
+
+* Adds support for subquery pushdown on reference tables
+
+* Adds shard pruning support for `IN` and `ANY`
+
+* Adds support for `UPDATE` and `DELETE` commands that prune down to 0 shard
+
+* Enhances transaction support by relaxing some transaction restrictions
+
+* Extends transaction support to more operations
+
+* Fixes a bug causing crash if distributed table has no shards
+
+* Fixes a bug causing crash when removing inactive node
+
+* Fixes a bug causing failure during `COPY` on tables with dropped columns
+
+* Fixes a bug causing failure during `DROP EXTENSION`
```

Comment:
Is this the issue introduced by distributed deadlock detection or another one?

Reply:
No, this was existing bug before deadlock detection.